aws_lambda_powertools/logging/formatters/datadog.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import annotations
+
+from typing import Any, Callable
+
+from aws_lambda_powertools.logging.formatter import LambdaPowertoolsFormatter
+
+
+class DatadogLogFormatter(LambdaPowertoolsFormatter):
+    def __init__(
+        self,
+        json_serializer: Callable[[dict], str] | None = None,
+        json_deserializer: Callable[[dict | str | bool | int | float], str] | None = None,
+        json_default: Callable[[Any], Any] | None = None,
+        datefmt: str | None = None,
+        use_datetime_directive: bool = False,
+        log_record_order: list[str] | None = None,
+        utc: bool = False,
+        use_rfc3339: bool = True,
```

Comment:
Not change required, just a question: the only difference between this constructor and the base LambdaPowertoolsFormatter is this `use_rfc3399` boolean that defaults to `true`, right?

Reply:
Would there be any downsides to making `use_rfc3399` default to true if it were communicated well enough in release notes or the next major version?